src/containers/sidebars/featured-map-card/featured-map-card-component.jsx

```diff
@@ -68,6 +128,17 @@ function FeaturedMapCardComponent({
                 <T _str={featuredMap.description} />
               </p>
             </section>
+            <section className={styles.buttonSection}>
+              <button
```

Comment:
Is it possible to use the existing button components for code and design coherence?